hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -378,7 +389,17 @@
            for (int j = 0; j < numMetaReplicas; j++) {
              RegionInfo meta = RegionReplicaUtil.getRegionInfoForReplica(
                                      RegionInfoBuilder.FIRST_META_REGIONINFO, j);
-             ServerName metaLocation = MetaTableLocator.waitMetaRegionLocation(master.getZooKeeper(), j, 1);
+             //If a metaLocation is null, All of its info would be empty here to be displayed.
+             ServerName metaLocation = null;
+             try {
+               metaLocation = MetaTableLocator.waitMetaRegionLocation(master.getZooKeeper(), j, 1);
+             } catch (NotAllMetaRegionsOnlineException e) {
+               //Since a region in transition state throw a NotAllMetaRegionsOnlineException then
```

Comment:
@Apache9 What about this explanation here? By the way, I added a 'continue' here to skip to display rit region. IMHO, we do not need to show region info here since the home page of master status UI has show which region is in transition and why